src/Qir/Execution/Tools/Microsoft.Quantum.Qir.Tools.csproj

```diff
@@ -67,7 +67,7 @@
 
   <ItemGroup>
     <None Include="$(QSimDll)">
-      <Link>Libraries\%(FileName)%(Extension)</Link>
+      <Link>Libraries\win-x64\%(FileName)%(Extension)</Link>
```

Comment:
Why is this Windows specific? This looks wrong. What is testing this behavior?

Reply:
Simulator.Dev.Props has some good examples of how to handle simulator DLLs in a cross-platform way, may be interesting to look at as a reference.

Reply:
Since the dll being move is Windows specific, it needs to go in the Windows specific folder. If the dll is also copied in from the Native folder as performed by the following ItemGroup, it will override the dll copied in this ItemGroup. That is only the case on the ADO builds. However, on local builds we need this ItemGroup's copy to make sure that we have the dll in a place the tool can find. Previously the dll was placed in directly in the Libraries folder, which was non-specific to a platform. This caused errors during Linux and Mac ADO builds because it was overriding the platform-appropriate dll with the one for Windows.

Reply:
I think that makes sense, but that then leaves the problem of getting the native shared libraries for the macOS and Linux runtimes into the tool package. You should be able to pull those out from the `xplat` artifact during e2e builds, with per-repo builds falling back to Windows-only.

Reply:
I'm not sure I understand the problem you're suggesting. The natvie shared libraries for the macOS and Linux runtimes are in the tool package.

Reply:
Resolved offline. The itemgroup now is specific to the OS being build on.